docs/source/06_resources/01_faq.md

```diff
@@ -51,6 +33,13 @@ The primary differences to Bonobo ETL and Bubbles are related to the following f
  - **Automatic dependency resolution for pipelines**. The `Pipeline` module also maps out dependencies between nodes and displays the results of this in a sophisticated but easy to understand directed acyclic graph.
  - Extensibility
 
+## How is Kedro similar to other projects?
+
+* Kedro's pipeline syntax is absolutely minimal (even supporting lambdas for simple transitions), and is inspired by the [Clojure library core.graph](https://github.com/plumatic/plumbing)
```

Comment:
Do you have a reference regarding Kedro being inspired by `Clojure library core.graph`?